src/components/Stories/components/VideoView/VideoView.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import {block} from '../../../utils/cn';
+import {StoryMedia} from '../../types';
+
+import './VideoView.scss';
+
+const b = block('stories-video-view');
+
+export interface VideoViewProps {
+    media: StoryMedia;
+}
+
+export function VideoView({media}: VideoViewProps) {
+    return (
+        <video
+            className={b()}
```

Comment:
What about having `poster` here? Video is displaying black square initially instead